src/main/kotlin/net/atos/zac/enkelvoudiginformatieobject/model/EnkelvoudigInformatieObjectLock.kt

```diff
@@ -0,0 +1,44 @@
+/*
+ * SPDX-FileCopyrightText: 2022 Atos
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+package net.atos.zac.enkelvoudiginformatieobject.model
+
+import jakarta.persistence.Column
+import jakarta.persistence.Entity
+import jakarta.persistence.GeneratedValue
+import jakarta.persistence.GenerationType
+import jakarta.persistence.Id
+import jakarta.persistence.SequenceGenerator
+import jakarta.persistence.Table
+import jakarta.validation.constraints.NotBlank
+import net.atos.zac.util.FlywayIntegrator
+import nl.lifely.zac.util.AllOpen
+import nl.lifely.zac.util.NoArgConstructor
+import java.util.UUID
+
+@Entity
+@Table(schema = FlywayIntegrator.SCHEMA, name = "enkelvoudiginformatieobject_lock")
+@SequenceGenerator(
+    schema = FlywayIntegrator.SCHEMA,
+    name = "sq_enkelvoudiginformatieobject_lock",
+    sequenceName = "sq_enkelvoudiginformatieobject_lock",
+    allocationSize = 1
+)
+@AllOpen
+@NoArgConstructor
+class EnkelvoudigInformatieObjectLock {
+    @Id
+    @GeneratedValue(generator = "sq_enkelvoudiginformatieobject_lock", strategy = GenerationType.SEQUENCE)
+    @Column(name = "id_enkelvoudiginformatieobject_lock")
+    var id: Long = 0
+
+    @Column(name = "uuid_enkelvoudiginformatieobject", nullable = false)
+    lateinit var enkelvoudiginformatieobjectUUID: UUID
```

Comment:
Why `lateinit` here? That seems a bit weird for such a class. 
```suggestion
    var enkelvoudiginformatieobjectUUID: UUID? = null
```
or preferably a val but not sure if that works at all?

Reply:
My idea was to be sure that we always have a value as the column is not nullable

Reply:
Normally you then use a `val` or actually: a `data class` but I guess that does not work with these entity classes? I'm not sure what is best practise here to be honest.